### ISSUE TITLE
Help fine-tuning on other CT data

### ISSUE THREAD

**Issue by AgustinaLaGreca** [MrGiovanni/ModelsGenesis#39]
Hello @MrGiovanni, thanks a lot for your work! It is very cool and interesting. 

I am working on image classification (disease positive vs disease negative) in CT of head and neck cancer. The problem is that, as you state, the genesis chest model was trained on the -1000 HU to 1000 HU, and my interesting range would be much less (approx -50 HU to 200 HU). My dataset size is rather limited (less than 600 patients) so that is why I think a pre-trained model is a good approach but I was wondering if you have any advice on this problem or any data normalization technique. Thanks a lot in advance, 

Best, 

Agustina

**Comment by MrGiovanni** [comment]
Hi @AgustinaLaGreca 

Thanks for reaching out. If your interesting range is -50 to 200 HU, I think it is still okay to use Models Genesis because we observe performance gain in CTPA and FLAIR imaging modalities, which are very different from the original CT intensity values.

Just make sure you normalize the input between [0, 1] by

```
hu_min, hu_max = -50, 200
image[image < hu_min] = hu_min
image[image > hu_max] = hu_max
image = (image - hu_min) / (hu_max - hu_min)
```

Thanks,

Zongwei

**Comment by AgustinaLaGreca** [comment]
Thanks a lot for the super fast reply @MrGiovanni !

Ok then I was on the right direction with the normalization. Also can I ask you if you have any tips on hyper-parameter search for fine-tuning? (size of fully-connected layers, number of frozen layers, etc.) Thanks a lot in advance!!

**Comment by MrGiovanni** [comment]
Hi @AgustinaLaGreca 

Sorry for the delay. No, I don't have a fancy hyperparameter search or optimizer for fine-tuning. Everything is the default in keras/pytorch setup ('Adam'). Every layer is trainable. The size of fully connected layers can be found at https://github.com/MrGiovanni/ModelsGenesis/tree/master/keras#3-fine-tune-models-genesis-on-your-own-target-task


Thanks,

Zongwei
